requirements/base.txt

```diff
@@ -239,7 +235,7 @@ drf-extra-fields==3.4.0
     # via -r requirements/base.in
 drf-nested-routers==0.94.1
     # via commonground-api-common
-drf-spectacular[sidecar]==0.27.2
```

Comment:
I thought this was an issue for a second, but apparently it's just a visual thing, because the optional dependencies are added anyway: https://github.com/astral-sh/uv/issues/2163#issuecomment-1977054461